src/number_theory/cyclotomic/discriminant.lean

```diff
@@ -134,6 +134,73 @@ begin
   { apply_instance }
 end
 
+/-- If `p` is a prime and `is_cyclotomic_extension {p ^ (k + 1)} K L`, then the discriminant of
+`hζ.power_basis K` is `(-1) ^ ((p ^ (k + 1).totient) / 2) * p ^ (p ^ k * ((p - 1) * (k + 1) - 1))`
```

Comment:
```suggestion
`hζ.power_basis K` is `(-1) ^ ((p ^ (k + 1)).totient / 2) * p ^ (p ^ k * ((p - 1) * (k + 1) - 1))`
```
and would it be preferable to expand `(p ^ (k + 1)).totient` as `p^k * (p-1)`?

Reply:
I've aded `discr_prime_pow_ne_two'`: I prefer to keep both versions, since using `nat.totient` is the standard way of stating the result, that also holds for `n` composite. (Also I think `discr_prime_pow_ne_two'` is false in the limit case when the exponent is `0`.)